ui/js/graphs.js

```diff
@@ -242,6 +242,14 @@ perf.controller('GraphsCtrl', [
         series.flotSeries.points.show = series.visible;
         series.flotSeries.lines.show = false;
       });
+      
+      $("#overview-plot").bind("plotunselected",function() {
```

Comment:
Thanks @akhileshpillai. Minor nit, I think there should be a space after `plotunselected,`.